### ISSUE TITLE
Can't sign in to Google Account when using Advanced Protection Program

### ISSUE THREAD

**Issue by ATGardner** [MohGovIL/hamagen-react-native#149]
I am running the Android version 1.1.2, from the Play Store, on Pixel 3 with Android 10.

I have the Google Advanced Protection Program enabled with my Google Account.
When trying to sign into the account, in order to extract my Location History data, the browser window shows a message which says "This browser does not support security keys."
If I tap on "Try another way", I have only the "Security Key" option available (since I have the APP enabled). So currently I can't seem to use this new feature.

![image](https://user-images.githubusercontent.com/538720/78164297-3b583c00-7452-11ea-8448-82e5db1a4a38.png)
![image](https://user-images.githubusercontent.com/538720/78164356-4f9c3900-7452-11ea-8bd7-8a66b8003162.png)

**Comment by DanielMOH** [comment]
Thanks for your feedback! It helps us improve the app.
I admit that we haven't tested the app with Google Advanced Protection Program enabled, so we didn't encounter this issue. Your feedback will help us investigate it and try to find a solution.
Note that for the login to your Google account, the app uses the native WebView, so it seems like an inherent limitation of Android WebView with Google's own Advanced Protection Program, not unique to our app.
However, we'll try to further investigate it and look for a solution or at least a workaround.

**Comment by ATGardner** [comment]
Yes, I assumed this would not be a top-priority bug, and as you said - it is probably related to the WebView itself. But I just wanted to get the issue posted. Maybe somebody will know of some workaround or solution for this.
